tpot/base.py

```diff
@@ -584,7 +572,13 @@ def predict(self, features):
         """
         if not self._fitted_pipeline:
             raise RuntimeError('A pipeline has not yet been optimized. Please call fit() first.')
-        return self._fitted_pipeline.predict(features.astype(np.float64))
+
+        features = features.astype(np.float64)
+
+        if np.any(np.isnan(features)):
+            features = self._impute_values(features)
```

Comment:
When predicting, TPOT should use the same Imputer that it trained on the training data. So TPOT needs to internally store the trained Imputer from the `fit` call.